### PR TITLE
Update Golem-Verificator to 1.6.3.2

### PR DESCRIPTION
**Pull request by weaselix**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ eth-utils==1.0.3
 ethereum==1.6.1
 Golem-Messages==2.24.2
 Golem-Smart-Contracts-Interface==1.6.1
-Golem-Verificator==1.6.3
```
Comment (shadeofblue) [inline requirements.txt:37]: why is the version name so strange?

Comment (shadeofblue) [reply requirements.txt:37]: + shouldn't the requirements-to-freeze also be updated?

Comment (weaselix) [reply requirements.txt:37]: >  is the version name so strange?

We (@magdasta and me) decided that it is the best name in that situation: it was branched from 1.6.3 and there already exists version 1.6.3.1

Comment (shadeofblue) [reply requirements.txt:37]: curiouser and curiouser...

**Comment by shadeofblue** [comment]
anyway @rkondziolka pls, update `requirements-to-freeze` too

**Comment by shadeofblue** [comment]
@rkondziolka @magdasta please ensure the docker image is uploaded first - if it is, I can accept the pull request

**Comment by shadeofblue** [comment]
okay, it requires testing, which in turn, requires the pip package to be uploaded ... please merge only after tests confirm that it fixes the issue

**Review by mfranciszkiewicz — DISMISSED** [review]
The Docker image cannot be based on 1.12, please change the tag and upload the image as 1.8.1.

1.12 uses a different entrypoint.

**Comment by mfranciszkiewicz** [comment]
I've uploaded the wheel, feel free to remove it and deploy your own package if needed

**Comment by ederenn** [comment]
```
WARNING  [golem.task.taskthread              ] Task computing error
WARNING  [golem.task                         ] Task not tested properly 500 Server Error: Internal Server Error ("VolumeDriver.Mount: exit status 255")
WARNING  [golem.task.rpc                     ] Test task error "{'name': 'scene-Helicopter-27-cycl', 'resources': ['C:\\Users\\ederenn\\Desktop\\scene-Helicopter-27-cycles.blend'], 'compute_on': 'cpu', 'type': 'Blender', 'subtasks_count': 1, 'concent_enabled': False}": ('500 Server Error: Internal Server Error ("VolumeDriver.Mount: exit status 255")',)
```

when uploading helicopter.bledn file from interface

**Comment by mfranciszkiewicz** [comment]
@ederenn This looks like some other bug. Could you please test this on Linux instead?

**Comment by ederenn** [comment]
on linux verification ends in fail for every node

```
2019-02-01 15:22:56 INFO     apps.blender                        Crop no [0] rendered for verification. Time spent: 2.416548013687134.
2019-02-01 15:22:56 WARNING  golem.task.taskthread               Task computing error
2019-02-01 15:22:56 INFO     apps.blender                        Crop no [1] rendered for verification. Time spent: 2.314673900604248.
2019-02-01 15:22:56 WARNING  golem.task.taskthread               Task computing error
2019-02-01 15:22:57 INFO     apps.blender                        Crop no [2] rendered for verification. Time spent: 2.4357190132141113.
2019-02-01 15:22:57 WARNING  golem.task.taskthread               Task computing error
2019-02-01 15:22:57 WARNING  apps.blender                        Unexpected verification output for subtask '766773a2-262c-11e9-8d2f-6c15fa96c125',
2019-02-01 15:22:57 WARNING  apps.blender                        Failure callback <twisted.python.failure.Failure builtins.bool: False>
2019-02-01 15:22:57 WARNING  apps.core.verification_queue        Finishing verification with fail
```

**Comment by weaselix** [comment]
> The Docker image cannot be based on 1.12, please change the tag and upload the image as 1.8.1.
> 
> 1.12 uses a different entrypoint.

@mfranciszkiewicz image_metrics:1.8.1 is uploaded to dockerhub

**Review by kmazurek — APPROVED** [review]

**Pull request merged**